packages/shared-ux/page/solution_nav/src/with_solution_nav.tsx

```diff
@@ -53,20 +62,27 @@ export const withSolutionNav = (WrappedComponent: ComponentType<KibanaPageTempla
     const templateToUse = isEmptyState && !template ? 'centeredContent' : template;
 
     const pageSideBar = (
-      <KibanaPageTemplateSolutionNav
+      <SolutionNav
         isOpenOnDesktop={isSideNavOpenOnDesktop}
         onCollapse={toggleOpenOnDesktop}
         {...solutionNav}
       />
     );
+
     const pageSideBarProps = {
-      paddingSize: 'none',
+      paddingSize: 'none' as 'none',
       ...props.pageSideBarProps,
       className: sideBarClasses,
```

Comment:
Are the styles associated with the classes still in a different package? Feels odd to split them up.

Reply:
That's a good point.  I'll likely need to duplicate/move them.  I'll update this.